test/gui/shared/steps/steps.py

```diff
@@ -1299,11 +1299,7 @@ def step(context):
 @Then('the sync all checkbox should be checked')
 def step(context):
     newAccount = AccountConnectionWizard()
-    test.compare(
-        waitForObjectExists(newAccount.SYNC_DIALOG_ROOT_FOLDER).checkState,
-        "checked",
-        "Assert sync all checkbox is checked",
-    )
+    waitForObject(newAccount.SYNC_CONNECTION_WIZARD_FINISH_BUTTON)
```

Comment:
does checking the visibility of finish button satisfies the statement `Then the sync all checkbox should be checked`? 🤔